src/optimize/base_optimizer.js

```diff
@@ -389,7 +384,7 @@ export default class BaseOptimizer {
       },
 
       resolve: {
-        extensions: ['.js', '.json'],
+        extensions: ['.js', '.jsx', '.ts', '.tsx', '.json'],
```

Comment:
Adding `.jsx` causes webpack to do extra lookups that are not necessary since use of `.jsx` is super uncommon and we haven't supported it for a while.

Reply:
👍